README.md

```diff
@@ -62,6 +62,14 @@ $ export NR_NATIVE_METRICS_REMOTE_PATH=path/to/download/folder/
 $ npm install @newrelic/native-metrics
 ```
 
+You can also specify a proxy host to route requests through using the `NR_NATIVE_METRICS_PROXY_HOST` environment variable:
+
+```sh
+$ export NR_NATIVE_METRICS_NO_BUILD=true
```

Comment:
i know this PR was to address the ask to support proxy servers but feels backwards to favor building locally over downloading.  In this new use case you'll have to specify 2 env vars instead of just 1

Reply:
Do you think we should change the default, from preferring build to preferring download? Let's file another issue for it.